services/wallet/commands.go

```diff
@@ -830,6 +852,19 @@ func (c *controlCommand) Run(parent context.Context) error {
 	return err
 }
 
+func (c *controlCommand) NewError(err error) bool {
```

Comment:
Why 3 exactly? could you add a description of the behavior that we want to achieve please?

Reply:
sure

Reply:
3 is arbitrary here

Reply:
@cammellos added two sentences to PRs description, let me know if that's enough